pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -292,6 +292,40 @@ class Build {
 
         return jdkRepo
     }
+
+    private String allocateEBCnodesForTest(testType) {
+        def test_target = buildConfig.TARGET_OS + '_' + buildConfig.ARCHITECTURE
+        // All targets are not supported via EBC yet. We need to test that target and add to default.json later on to make it enable
+        if ( ! DEFAULTS_JSON['testDetails']['ebcEnabledTargets'][test_target].find { it == testType } ) {
+            context.println "EBC does not support for $test_target->$testType yet!"
+            return ''
+        }
+
+        // context.library 'jenkins-ci-websphere' 
+        def group_label_UUID = 'Adopt-semeru_test_machine_' + UUID.randomUUID().toString()
+        def num_machines = '0'
+        def test_index = DEFAULTS_JSON['testDetails']['defaultDynamicParas']['testLists'].indexOf(testType)
+        num_machines = DEFAULTS_JSON['testDetails']['defaultDynamicParas']['numMachines'].get(test_index)
+        def nodes_ebc_args = new Object[num_machines]
+
+        if (num_machines == '0' ) {
+            num_machines = '1'
+        }
+
+        def jobName = '/EBC_Create_Node'
+        def ebcJob = context.build job: jobName,
+            propagate: false,
+            wait: true,
+            parameters: [
+                    context.string(name: 'group_label', value: group_label_UUID),
+                    context.string(name: 'ebcPlan', value: 'hur-runtimesJenkins-ubuntu22_x86.yml'),
```

Comment:
This is ok for now, but only allows us to enable xlinux. We should just add it to defaults.json now.

Reply:
Right. It only shows the fellow for now. But in future we can push this login either in EBC create node job or expand it here base on job parameters.

Reply:
As per our dicussion, perhaps a plan name of "semeru-test-$platform" here would be a good solution for now until we need to expand it.

Reply:
Yes. Another approach could be call with some public inputs that can be used the same with other usages then in the EBC pipeline we will decide about plan.